.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,5 @@ Depending on what your PR does, here are a few things you might want to address
 + [ ] what are the (breaking) changes that this PR makes?
 + [ ] important background, or details about the implementation
 + [ ] are the changes—especially new features—covered by tests and docstrings?
-+ [ ] [linting/style checks have been run](https://docs.pymc.io/en/latest/contributing/python_style.html)
```

Comment:
why remove this?

Reply:
Because they run automatically in the CI.
Ticking the box or not doesn't make a difference. What counts is `pre-commit` being ❌|✔.

Reply:
Maybe we could add a link to the checklist instead? As you mentioned before, 2% of users will have read the contributing guide, but maybe 20% read the pull request template and will benefit from the reference/extra context

Reply:
I added the link above the checklist.